content/03.prior-knowledge.knowledge-graphs.md

```diff
@@ -0,0 +1,13 @@
+### Knowledge graphs for rare disease
+
+An intrinsic constraint in the study of rare disease is the availability of large, normalized datasets. 
+This limits our ability to study genotype-phenotype relationships or other key attributes of rare diseases. 
+A potentially powerful strategy for evaluating genotype-phenotype relationships or discovering repurposable drugs when large datasets are scarce or nonexistent is to develop and use knowledge graphs.
+Knowledge graphs integrate related-but-different data types, creating a rich and complex data source. 
+Examples of well-known public biomedical knowledge graphs and graph frameworks that could be useful in the rare disease context include the Monarch Graph Database[@doi:10.1093/nar/gkw1128], hetionet[@doi:10.7554/eLife.26726], and PheKnowLator[@doi:10.1101/2020.04.30.071407]. 
+These graphs connect information like genetic, functional, chemical, clinical, and ontological data to enable the end user to explore many types of data and their relationships with disease phenotypes whether through manual review[@doi:10.1093/database/baaa015] or computational methods. 
```

Comment:
requesting citation for the computational method bit, otherwise this looks good.